packages/vue-dot/src/patterns/FormField/fields/ChoiceSliderField.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+	<VSlider
+		v-if="field.items"
+		color="accent"
+		:value="getIndex(field.value)"
+		class="mt-5"
+		v-bind="field.metadata"
+		:thumb-label="thumbLabel"
+		:tick-labels="thickLabels"
+		:max="field.items.length -1"
+		@change="valueChanged"
+	>
+		<template
+			v-if="thumbLabel"
+			#thumb-label="props"
+		>
+			<span>
+				{{ labels[props.value] }}
+			</span>
+		</template>
+	</VSlider>
+</template>
+
+<script lang="ts">
+	import Vue from 'vue';
+	import Component from 'vue-class-component';
+
+	import FieldComponent from '../mixins/fieldComponent';
+	import { FieldValue } from '../types';
+
+	/** Form field to enter text */
+	@Component
+	export default class ChoiceSliderField extends FieldComponent {
+		/** get the index of the selected item by value */
```

Comment:
```suggestion
		/** Get the index of the selected item by value */
```
Est-ce que tu peux décrire le paramètre et le type de retour ? C'est un peu verbeux mais je pense que c'est plus explicite, par exemple :+1: 
```ts
/**
 * Get the index of the selected item by value
 *
 * @param {FieldValue} value Description
 * @return {type} Description
 */
```